sdks/python/apache_beam/metrics/execution.py

```diff
@@ -36,7 +36,9 @@
 
 
 class MetricKey(object):
-  """Key used to identify instance of metric cell.
+  """For internal use only; no backwards-compatibility guarantees.
```

Comment:
Should we just update the comment at the top of the file that says `Internal classes for Metrics API.`